Lib/logging/__init__.py

```diff
@@ -339,9 +339,15 @@ def __init__(self, name, level, pathname, lineno,
         self.stack_info = sinfo
         self.lineno = lineno
         self.funcName = func
-        self.created = ct
-        self.msecs = int((ct - int(ct)) * 1000) + 0.0  # see gh-89047
-        self.relativeCreated = (self.created - _startTime) * 1000
+        self.created = ct / 1e9  # ns to float seconds
```

Comment:
Do we also want to add `self.created_ns` as a public attribute?

Reply:
There hasn't been a need for it thus far, and it's not necessary to fix the floating point representation issue. So I'd suggest not.